src/core/exec-invoke.c

```diff
@@ -2548,6 +2557,173 @@ static int create_many_symlinks(const char *root, const char *source, char **sym
         return 0;
 }
 
+static int set_exec_storage_quota(int fd, uint32_t proj_id, const QuotaLimit *ql) {
+        int r;
+        uint64_t block_limit = 0, inode_limit = 0;
+
+        assert(fd >= 0);
+        assert(ql);
+
+        if (ql->quota_absolute == 0 || ql->quota_scale == 0)
+                block_limit = 1, inode_limit = 1;
```

Comment:
we never use `,` outside of for loop expressions, please don't start.

just rewrite to:

```c
block_limit = inode_limit = 1;
```